week03/html/contact.html

```diff
@@ -0,0 +1,51 @@
+<link href="../css/contact.css" rel="stylesheet" /> 
+<link rel="icon" type="image/png" sizes="16x16" href="../src/billyLogo.png"/>
+
+<title>Lee Gabin</title>
+
+<body>
+    <!--상단바-->
+
+    <header id="header">
+        <div class="navbar">
+        <div class="nav1">
+        <a href="../html/home.html" class="logo">
+        <img src="../src/mainLogo.png" height="70px" />
+    </a>
+    </div>
+    <div class="nav2"></div>
+    <div class="nav3">
+    <ul>
+        <li><a href="../html/home.html">Home</a></li>
+        <li><a href="../html/about.html">About</a></li>
+        <li><a href="../html/project.html">Project</a></li>
+        <li><a href="../html/activity.html">Activity</a></li>
+        <li><a href="../html/contact.html" style="color: black;">Contact</a></li>
+    </ul>
+    </div>
+    </div>
+    </header>
+
+    <!--내용-->
+    <div class="contactMe">
+        <div class="title">Contact</div>
+        <div class="mail">
+            <a class="t1">E-MAIL</a><br>
+            <a href="mailto: gb94@ewhain.net">
+            <img href="mailto: gb94@ewhain.net" src="../src/gmailLogo.png" height="150px"></a>

```

Comment:
1. `img` 태그에는 `href` 속성이 없습니다.
2. `mailto:` 띄어쓰기 없이 바로 이메일을 적어야 합니다.
3. 태그를 잘 닫아주어야 합니다.
4. 인라인 스타일 또는 외부 스타일로 통일하는 것이 좋습니다.

```suggestion
            <a href="mailto:gb94@ewhain.net"><img src="../src/gmailLogo.png" /></a>
```